waspc/src/Wasp/Generator/AuthProviders.hs

```diff
@@ -0,0 +1,33 @@
+module Wasp.Generator.AuthProviders where
+
+import StrongPath (relfile)
+import qualified Wasp.AppSpec.App.Dependency as App.Dependency
+import qualified Wasp.Generator.AuthProviders.Local as L
+import qualified Wasp.Generator.AuthProviders.OAuth as OA
+
+googleAuthInfo :: OA.OAuthAuthInfo
```

Comment:
We define all of the provider info in a single place, it should hopefully make the whole experience of setting up a new provider more declarative than before.